Sources/Kommunicate/Classes/KMPushNotificationHandler.swift

```diff
@@ -39,17 +39,19 @@ public class KMPushNotificationHandler: Localizable {
             else { return }
 
             guard let userInfo = notification.userInfo as? [String: Any], let state = userInfo["updateUI"] as? NSNumber else { return }
+            
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.3) {
```

Comment:
Why are we adding this?

Reply:
when app is opened through tapping notification,` ALPushAssist().topViewController` is nil. So couldn't present  the conversation screen. This is happening only when app is not running, when the notification tapped. Thatswhy I have added this.

Reply:
ok